modules/ROOT/pages/overview-of-bonita-bpm-in-a-cluster.adoc

```diff
@@ -115,3 +115,22 @@ There are some drawbacks to the basic Bonita in a cluster configuration, but the
 * Only active-active modes is supported. There is no mechanism for starting a new node if a cluster node goes out of service. This could be provided by a third-party application.
 * There is no integrated load balancer, so the client application selects the node on which to make a API call. You should use an external load balancer. For high volume, a hardware load balancer is recommended.
 * All the nodes must use the same database, so the database itself must be highly available.
+
+=== Upgrading the version of a Bonita Platform in cluster
+
+[WARNING]
+====
+
+All nodes of the cluster must use the *exact* same version. It will not be possible to start a node on an existing cluster having a different version.
+====
+
+Since all nodes must use the exact same version of the Bonita Platform, the cluster must be shut down entirely to be able to change the version.
```

Comment:
```suggestion
Since all nodes must use the exact same version of the Bonita Runtime, the cluster must be shut down entirely to be able to change the version.
```